mage/query-modules/python/meta-util.md

```diff
@@ -0,0 +1,398 @@
+---
+title: meta_util
+sidebar_label: meta_util
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
```

Comment:
E za sto je ovo molim te, bas sam se pital anedavno cemu to sluzi?

Reply:
A custom component that gives you background color (highlights text) in a table explaining the algorithm. In this case, the color is #FB6E00 (orange), but the component can be used in other places in different colors.
<img width="342" alt="Screenshot 2023-01-30 at 13 31 40" src="https://user-images.githubusercontent.com/61758502/215477759-d9996d1e-852b-44c8-9b5c-e5dbbd14b65e.png">